targets/ramips-mt7621

```diff
@@ -149,6 +149,11 @@ device('zyxel-nwa55axe', 'zyxel_nwa55axe', {
 	broken = true, -- Missing LED / Reset button
 })
 
+device('zyxel-wsm20', 'zyxel_wsm20', {
+	factory = false,
+	aliases = {'zyxel-multy-m1'},
```

Comment:
Is this alias needed?

Reply:
No, as [per documentation](https://gluon.readthedocs.io/en/latest/dev/hardware.html#aliases-and-manifest-aliases) it only makes it easiert to find the image.

Reply:
I think that's not necessary here. For example, with the Aruba AP-303 and the Aruba Instant On AP11 from the docs, the same hardware can be found in different packages, with different firmwares sold at different price points.

"Multi M1" is just a name given by Zyxel marketing, which they probably considered catchier than the product number WSM20. It's not a separate product with a different name on the case depending on what you buy.